### DISCUSSION TITLE
TW colors?

### DISCUSSION THREAD

**Discussion by fpolli** [L-Blondy/tw-colors#50] [Q&A] [answered]
Can you use tailwind colors in the theme? For example:

+        createThemes({
+           light: { 
+              'primary': colors.blue[400],
+              'secondary': colors.blue[900],
+              'brand': colors.indigo[600]
+              // ...other colors
+           }
+        })

Thanks

**Comment by L-Blondy** [comment, marked as answer]
Yes you can
